pywinauto/timings.py

```diff
@@ -319,11 +319,45 @@ def WaitUntil(
             
     return func_val
 
+#=========================================================================
+def wait_until(

```

Comment:
Basically, we try to convert all the methos to PEP-8 compatible names and to keep the old names as aliases. So ```WaitUntil``` should be renamed to ```wait_until```
```python
def wait_until():
    # here comes the code
    pass
# add the alias for the backward compatibility
WaitUntil = wait_until 

# Here we come up with a nice decorator name 
def nice_decorator_name_for_wait_until_decorator
```